staging/vhost-device-sound/Cargo.toml

```diff
@@ -26,12 +26,12 @@ thiserror = "1.0"
 vhost = { version = "0.8", features = ["vhost-user-slave"] }
 vhost-user-backend = "0.10"
 virtio-bindings = "0.2.1"
-virtio-queue = "0.9"
-vm-memory = "0.12"
+virtio-queue = "0.10"
+vm-memory = "0.13"
```

Comment:
Should we use `0.13.1`?

Reply:
Yes... That one brings the missing traits. Will update